src/api/ledger/balance-sheet.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const _ = require('lodash');
+const utils = require('./utils');
+const validate = utils.common.validate;
+const composeAsync = utils.common.composeAsync;
+const convertErrors = utils.common.convertErrors;
+
+function formatBalanceSheet({balances, obligations, assets}) {
+  const result = {};
+
+  if (!_.isUndefined(balances)) {
+    result.balances = Object.keys(balances).map((k) => {
+      return {
+        counterparty: k,
+        balances: balances[k]
+      };
+    });
+  }
+  if (!_.isUndefined(assets)) {
+    result.assets = Object.keys(assets).map((k) => {
+      return {
+        counterparty: k,
+        assets: assets[k]
+      };
+    });
+  }
+  if (!_.isUndefined(obligations)) {
+    result.obligations = Object.keys(obligations).map((k) => {
```

Comment:
lodash map gives you both value and key, so you could do `result.obligations = _.map(obligations, (value, currency) => {currency, value})`, and similar for balances and assets